### PR TITLE
Authorization - intros and headings

### PR DESCRIPTION
**Pull request by johnpapa**
added intros for heading sections

needs review before #sign-off

### CODE REVIEW

**Comment by herohua** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs-ref-services/authorization.md](https://github.com/MicrosoftDocs/azure-docs-sdk-node/blob/johnpapa-patch-3/docs-ref-services/authorization.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2017%5C7%5C18%5Cb6261451-6613-d51e-95b5-8b5dc96f3a2c%5CPullRequest%5C201707180004507221-104%5Cworkflow_report.html).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by herohua** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs-ref-services/authorization.md](https://github.com/MicrosoftDocs/azure-docs-sdk-node/blob/johnpapa-patch-3/docs-ref-services/authorization.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2017%5C7%5C18%5Cb6261451-6613-d51e-95b5-8b5dc96f3a2c%5CPullRequest%5C201707180010056830-104%5Cworkflow_report.html).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by herohua** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs-ref-services/authorization.md](https://github.com/MicrosoftDocs/azure-docs-sdk-node/blob/johnpapa-patch-3/docs-ref-services/authorization.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2017%5C7%5C18%5Cb6261451-6613-d51e-95b5-8b5dc96f3a2c%5CPullRequest%5C201707180051463050-104%5Cworkflow_report.html).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: docs-ref-services/authorization.md [hunk 1]
```diff
@@ -16,30 +16,40 @@ ms.service: Authorization
...
 Azure App Service Authentication / Authorization is a feature that provides a way for your application to sign in users so that code doesn't have to be changed on the app backend. Authorization provides an easy way to protect your application and work with per-user data.
 
 ## Management
 
 Use npm to install the Azure Authorization modules for Node.js
 
-### Install the modules with npm
+### Install npm module
+
+Install the Azure authorization npm module
```
Comment (clarkio) [inline docs-ref-services/authorization.md]: Capitalize `Authorization`

File: docs-ref-services/authorization.md [hunk 2]
```diff
@@ -16,30 +16,40 @@ ms.service: Authorization
...
+
 ```bash
 npm install azure-arm-authorization
 ```
+
 ### Example
+
+This example lists all role assignments for the requested resource group.
```
Comment (clarkio) [inline docs-ref-services/authorization.md]: Other docs don't have the `.` Should this be removed?

**Comment by herohua** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs-ref-services/authorization.md](https://github.com/MicrosoftDocs/azure-docs-sdk-node/blob/johnpapa-patch-3/docs-ref-services/authorization.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2017%5C7%5C18%5Cb6261451-6613-d51e-95b5-8b5dc96f3a2c%5CPullRequest%5C201707180407009368-104%5Cworkflow_report.html).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: docs-ref-services/authorization.md [hunk 3]
```diff
@@ -13,33 +13,43 @@ ms.devlang: nodejs
...
 const msRestAzure = require('ms-rest-azure');
 const authorizationManagement = require('azure-arm-authorization');
 
-msRestAzure.interactiveLogin( (err, credentials) => {
- const client = new authorizationManagement(credentials, 'your-subscription-id');
- const resourceGroupName = 'resource-group-name';
+const resourceGroup = 'resource-group-name';
+const subscriptionId = 'your-subscription-id';
+
+msRestAzure.interactiveLogin((err, credentials) => {
```
Comment (jawache) [inline docs-ref-services/authorization.md]: Use promise vs callback

File: docs-ref-services/authorization.md [hunk 4]
```diff
@@ -13,33 +13,43 @@ ms.devlang: nodejs
...
+ const client = new authorizationManagement(credentials, subscriptionId);
  client.roleAssignments.listForResourceGroup(resourceGroupName, (err, result) => {
```
Comment (jawache) [inline docs-ref-services/authorization.md]: Use promise vs callback

**Comment by johnpapa** [comment]
#sign-off LGTM

**Comment by herohua** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs-ref-services/authorization.md](https://github.com/MicrosoftDocs/azure-docs-sdk-node/blob/johnpapa-patch-3/docs-ref-services/authorization.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2017%5C7%5C18%5Cb6261451-6613-d51e-95b5-8b5dc96f3a2c%5CPullRequest%5C201707181336430063-104%5Cworkflow_report.html).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Pull request merged**